omnibus/Gemfile.lock

```diff
@@ -465,9 +465,8 @@ GEM
 
 PLATFORMS
   ruby
-  x64-mingw
-  x64-mingw-ucrt
```

Comment:
@johnmccrae Kindly do not merge this pr. It seems you had some stashed changed from other branch and then  did a manual gem update which has resulted in it removing ucrt versions, this will break omnibus builds, https://github.com/chef/chef/pull/13443/commits.
The corresponding commits in this branch:
https://github.com/chef/chef/pull/13443/commits/748c4707b2d0d7767b59acb7ca19893dbe172ed9
https://github.com/chef/chef/pull/13443/commits/381faa761a68177cbe5d78ccb1f020da864f0595
whereas actual version bump this had is only this https://github.com/chef/chef/pull/13443/commits/853f3efd049dacb7c7c159509f95666715ac3242